elastic/kibana
# [Discover] Fix infinite scrolling using Classic table

## Summary

This PR solves an issue that infinite scrolling doesn't work when there's very little vertical screen space available for the container containing the document table. 

### Testing
1. Make sure you're using Classic table
2. Limit height of your browser to e.g. 700, and make sure sidebar is displayed, the bug doesn't happen in "mobile" view, which scrolls the whole page
3. Go to Discover
4. Without this PR, when scrolling down the document table, no more than 50 documents are displayed, with this PR, when scrolling down documents in patches of 50 are added to the table until it reaches the number of 500





### Checklist
